src/services/RelayService/RelayService.ts

```diff
@@ -35,7 +35,11 @@ class RelayService {
       requestMsg: {/* Empty Msg */}
     }
     // Call method through rpc
-    console.debug("Sending  request through rpcChannel")
+    console.info(
+      `Calling Relay ${methodName.replace(/_([a-z])/g, (_, group1) => group1.toUpperCase())}`+
+     ` through Rpc Channel..`
+     )
```

Comment:
so the original `method_name` is snake_case but you're logging it as camelCase? why?

Reply:
basically on the python server side it expects to get `method name` in snake_case but on this javascript client, camelCase is convention. So I think logging out the method name in snake_case is weird when the method that is being called on this side is in camelCase